API/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "whitethoothapi",
+  "version": "1.0.0",
+  "description": "Mock API for teams with multi environment support",
+  "main": "index.js",
+  "scripts": {
+    "start": "npm run build && node ./build/index.js",
```

Comment:
maybe the user would be able to set the port through a parameter for to start command (with default)?

Reply:
We will define dedicated task for all the startup of the full service